src/data/electronica-2020.json

```
@@ -0,0 +1,581 @@
+    "creditos": 2,
+    "correlativas": "IIE",
+    "categoria": "Materias Obligatorias",
+    "level": 4
+  },
+  {
+    "id": "CM",
+    "materia": "Circuitos Microelectrónicos",
+    "creditos": 6,
+    "correlativas": "IDE-AC",
+    "categoria": "Materias Obligatorias",
+    "level": 4
+  },
+  {
+    "id": "PE",
+    "materia": "Procesos Estocásticos",
+    "creditos": 6,
+    "correlativas": "PYE-SS",
+    "categoria": "Materias Obligatorias",
+    "level": 4
+  },
+  {
+    "id": "CA",
+    "materia": "Control Automáticos",
+    "creditos": 6,
+    "correlativas": "EMC-SS",
+    "categoria": "Materias Obligatorias",
+    "level": 4
+  },
+  {
+    "id": "TSE",
+    "materia": "Taller de Sistemas Embebidos",
+    "creditos": 6,
+    "correlativas": "AP-IDE-SD",
+    "categoria": "Materias Obligatorias",
+    "level": 4
+  },
+  {
+    "id": "QE",
+    "materia": "Química y Electroquímica",
+    "creditos": 6,
+    "correlativas": "EMC",
+    "categoria": "Materias Obligatorias",
+    "level": 5
+  },
+  {
+    "id": "EA",
+    "materia": "Electromagnetismo Aplicado",
+    "creditos": 4,
+    "correlativas": "AC",
+    "categoria": "Materias Obligatorias",
+    "level": 5
+  },
+  {
+    "id": "TAC",
+    "materia": "Taller de Automatización y Control",
+    "creditos": 6,
+    "correlativas": "AP-CA-SD",
+    "categoria": "Materias Obligatorias",
+    "level": 5
+  },
+  {
+    "id": "TPS",
+    "materia": "Taller de Procesamiento de Señales",
+    "creditos": 6,
+    "correlativas": "PE",
+    "categoria": "Materias Obligatorias",
+    "level": 5
+  },
+  {
+    "id": "DS",
+    "materia": "Dispositivos Semiconductores",
+    "creditos": 6,
+    "correlativas": "IDE-AC",
+    "categoria": "Materias Obligatorias",
+    "level": 6
+  },
+  {
+    "id": "TCD",
+    "materia": "Taller de Comunicaciones Digitales",
+    "creditos": 6,
+    "correlativas": "SS-RC",
+    "categoria": "Materias Obligatorias",
+    "level": 6
+  },
+  {
+    "id": "TDCE",
+    "materia": "Taller de Diseño de Circuitos Electrónicos",
+    "creditos": 6,
+    "correlativas": "CM-EA",
+    "categoria": "Materias Obligatorias",
+    "level": 6
+  },
+  {
+    "id": "HS",
+    "materia": "Higiene y Seguridad",
+    "creditos": 2,
+    "requiere": 100,
+    "requiereCBC": true,
+    "categoria": "Materias Obligatorias",
+    "level": 6
+  },
+  {
+    "id": "LEP",
+    "materia": "Legislación y Ejercicio Profesional",
+    "creditos": 2,
+    "requiere": 100,
+    "requiereCBC": true,
+    "categoria": "Materias Obligatorias",
+    "level": 7
+  },
+  {
+    "id": "GPE",
+    "materia": "Gestión de Proyectos Electrónicos",
+    "creditos": 6,
+    "correlativas": "PP",
+    "categoria": "Materias Obligatorias",
+    "level": 7
+  },
+  {
+    "id": "ISADS",
+    "materia": "Impacto Social, Ambiental y Desarrollo Sustentable",
+    "creditos": 4,
+    "requiere": 100,
+    "requiereCBC": true,
+    "categoria": "Materias Obligatorias",
+    "level": 8
```

Se que no es clavado lo que dice el plan, pero en otros planes del fiuba map siempre terminamos agrupando las ultimas materias para que quede mas uniforme el grafo

```suggestion
    "level": 7
```